models/netsuite2/netsuite2__balance_sheet.sql

```diff
@@ -72,19 +79,19 @@ balance_sheet as (
         end as converted_amount,
     
     case
-      when lower(accounts.type_name) = 'bank' then 1
-      when lower(accounts.type_name) = 'accounts receivable' then 2
-      when lower(accounts.type_name) = 'unbilled receivable' then 3
-      when lower(accounts.type_name) = 'other current asset' then 4
-      when lower(accounts.type_name) = 'fixed asset' then 5
-      when lower(accounts.type_name) = 'other asset' then 6
-      when lower(accounts.type_name) in ('deferred expense', 'prepaid expense') then 7
-      when lower(accounts.type_name) = 'accounts payable' then 8
-      when lower(accounts.type_name) = 'credit card' then 9
-      when lower(accounts.type_name) = 'other current liability' then 10
-      when lower(accounts.type_name) = 'long term liability' then 11
-      when lower(accounts.type_name) = 'deferred revenue' then 12
-      when lower(accounts.type_name) = 'equity' then 13
+      when lower(accounts.account_type_id) = 'bank' then 1
+      when lower(accounts.account_type_id) = 'acctrec' then 2
+      when lower(accounts.account_type_id) = 'unbilledrec' then 3
+      when lower(accounts.account_type_id) = 'othcurrasset' then 4
+      when lower(accounts.account_type_id) = 'fixedasset' then 5
+      when lower(accounts.account_type_id) = 'othasset' then 6
+      when lower(accounts.account_type_id) = 'deferexpense' then 7
```

Comment:
same here. `prepaid expense` removed since the id handles variations in the name.